Source/Csla.AspNetCore/Blazor/State/SessionManager.cs

```diff
@@ -0,0 +1,81 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SessionServerManager.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Manages all user session data</summary>
+//-----------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Csla.State;
+
+namespace Csla.Blazor.State
+{
+  /// <summary>
+  /// Manages all user session data for a given
+  /// root DI container.
+  /// </summary>
+  /// <param name="sessionIdManager"></param>
+  public class SessionManager(ISessionIdManager sessionIdManager) : ISessionManager
+  {
+    private readonly Dictionary<string, Session> _sessions = [];
```

Comment:
Isn't this instance always be on the server side? If so `Dictionary` is not thread-safe. Use the `ConcurrentDictionary` instead.